fuzzing/tools/launcher.py

```diff
@@ -35,7 +35,7 @@
 
 flags.DEFINE_integer(
     "timeout_secs",
-    20,
+    0,
```

Comment:
why 0? shouldn't regression shell script manually set to 0?

Reply:
ah 0 is continuous. This aligns with our current use, but I wonder if we should guarantee that the process should terminate. @stefanbucur ?

Reply:
That's a good question - but I'm not sure how this blocks the current PR? Can we revert this change and worry about it later?

Reply:
The original `timeout_secs` is set to 20, thus the user will have to set this flag explicitly to get a continuous fuzzing test.

Also, the `is_regression` flag is set to True in default, which means the remove in https://github.com/envoyproxy/envoy/pull/12530#discussion_r476725983 will make the `<name>_run` target becomes a regression test. Thus that comment is not fully resolved until it is using the new version of `bazel-rules-fuzzing` with this PR.